guides/common/modules/con_fapolicyd-on-server.adoc

```diff
@@ -0,0 +1,15 @@
+[id="fapolicyd-on-server_{context}"]
+ifeval::["{context}" == "{project-context}"]
+= Optional: fapolicyd on {ProjectServer}
+endif::[]
+ifeval::["{context}" == "{smart-proxy-context}"]
+= Optional: fapolicyd on {SmartProxyServer}
+endif::[]
+
+fapolicyd is a Linux kernel module that enforces file access policies.
+It provides an additional layer of security by monitoring and controlling access to files and directories on a system.
```

Comment:
I guess this will be required as an introductory part. 
System = You are having Project Server installed or going to install but with your suggestion, it's no longer required. :+1: